File: cpp/program/ProdukElektronik.cpp
#include "Produk.cpp"

class ProdukElektronik : public Produk {
protected:
    string merek;
    string model;
    string tegangan;

public:
    ProdukElektronik(string id = "", string n = "", string d = "", double h = 0,
                     string m = "", string mdl = "", string t = "")
        : Produk(id, n, d, h), merek(m), model(mdl), tegangan(t) {}

    string getMerek() const { return merek; }
    string getModel() const { return model; }
    string getTegangan() const { return tegangan; }
};

File: php/program/ProdukElektronik.php
<?php
require_once "Produk.php";

class ProdukElektronik extends Produk {
    protected $merek;
    protected $model;
    protected $tegangan;

    public function __construct($id = "", $nama = "", $deskripsi = "", $harga = 0,
                                $merek = "", $model = "", $tegangan = "") {
        parent::__construct($id, $nama, $deskripsi, $harga);
        $this->merek = $merek;
        $this->model = $model;
        $this->tegangan = $tegangan;
    }

    public function getMerek() { return $this->merek; }
    public function getModel() { return $this->model; }
    public function getTegangan() { return $this->tegangan; }
}
